src/main/java/io/confluent/connect/jdbc/JdbcUtils.java

```diff
@@ -154,5 +155,17 @@ public static String getAutoincrementColumn(Connection conn, String table) throw
   public static String formatUTC(Date date) {
     return DATE_FORMATTER.get().format(date);
   }
+
+  /**
+   * Get the string used for quoting identifiers in this database's SQL dialect.
+   * @param connection the database connection
+   * @return the quote string
+   * @throws SQLException
+   */
+  public static String getIdentifierQuoteString(Connection connection) throws SQLException {
+    String quoteString = connection.getMetaData().getIdentifierQuoteString();
+    quoteString = quoteString == null ? "" : quoteString;
```

Comment:
Yes, because this is always used to add quotes around identifiers, if the DB doesn't want quotes then it's harmless to use an empty string. It's easier to resolve the null issue here than have a bunch of null checks anywhere we use this.